src/App.css

```diff
@@ -325,6 +325,18 @@ body {
     url(./assets/background-science.jpg);
 }
 
+.app-banner-bg {
+  background-color: rgb(195, 30, 124) !important;
```

Comment:
Can you use an existing semantic color? It's important that we try to stay constrained to our color palette so that it doesn't become bloated.

Reply:
Wellll, I tried the semantic pink but it was soo bright. Let me try purple or violet. Or maybe teal..

Reply:
Ohhh ok, so I think it was never correctly applying the semantic "pink" css before. That's why I thought it looked ugly. I figured it out. I just have to add "pink" to `className` prop on my Segment